apps/judicial-system/api/src/app/modules/file/models/file.model.ts

```diff
@@ -10,7 +10,6 @@ import type {
 export class CaseFile implements TCaseFile {
   @Field(() => ID)
   readonly id!: string
-
```

Comment:
This was just to force the schema regeneration, devops is fixing so we don't need to do these random changes in the future to trigger it